WcaOnRails/config/routes.rb

```diff
@@ -58,7 +58,13 @@
   get 'polls/:id/vote' => 'votes#vote', as: 'polls_vote'
   get 'polls/:id/results' => 'polls#results', as: 'polls_results'
 
-  resources :teams, only: [:index, :new, :create, :update, :edit]
+  resources :committees do
+    resources :teams, only: [:new, :edit, :create, :update, :destroy] do
+      resources :team_members, only: [:new, :edit, :create, :update]
+    end
+    resources :committee_positions, only: [:index, :new, :edit, :create, :update, :destroy]
```

Comment:
This leads to weird url helpers (`committee_committee_positions_path`) and urls (`/committee/committee_positions`).
The solution is pretty simple: add two more options
`as: :positions` (for url helpers) and `paths: :positions` (for urls).